pykern/pkcli/github.py

```diff
@@ -171,14 +171,17 @@ def issue_update_alpha_pending(repo):
         sha='master',
         since=datetime.datetime.now() - datetime.timedelta(minutes=24 * 60),
     ):
-        m = re.search(r'#(\d+)', c.message)
+        m = re.search(r'([^\s]+\/.+)?#(\d+)', c.message)
```

Comment:
@rorour it looks like this contains the changes for your other PR. I think you should change the base of this branch to be master.

```sh
git checkout master
git pull
git checkout 120-add-to-gitignore
git rebase --onto master 80-parse-repo-with-issue
```